packages/hash/api/src/model/util.ts

```diff
@@ -330,6 +327,35 @@ export const generateSystemEntityTypeSchema = (
   };
 };
 
+export type EntityLinkTypeCreatorParams = {
+  entityLinkTypeId: VersionedUri;
+  title: string;
+  description: string;
+  actorId: string;
+};
+
+const entityLinkTypeUri: VersionedUri =
+  "https://blockprotocol.org/@blockprotocol/types/entity-type/link/v/1";
+
+/**
+ * Helper method for generating a link type schema for the Graph API.
+ *
+ * @todo make use of new type system package instead of ad-hoc types.
+ *   https://app.asana.com/0/1202805690238892/1202892835843657/f
```

Comment:
Should we remove this todo? It is referring to that commit SHA task :)

Reply:
Removed these in https://github.com/hashintel/hash/pull/1400/commits/cd81e53d78c6c463290e55b0731a604ff9d71611